src/server.py

```diff
@@ -45,7 +127,10 @@ async def redirect_link(request, link_endpoint):
                 )
             )
             url = await query.fetchone()
-            return response.redirect(url[1])
+            if url[3]:
+                return response.redirect(url[2])
```

Comment:
Isn't it safer to use something like this?

```suggestion
            if url.is_active:
                return response.redirect(url.url)
```

Reply:
Yes, thanks! 👍 I already had to change the index twice when changing DB columns, which wasn't very cash money. 💵